src/CMakeLists.txt

```diff
@@ -18,8 +22,23 @@ target_link_libraries(kdenow
     KF5::Service
 )
 
+target_link_libraries(kdenowd
+    kdenowcore
+)
+
+generate_export_header(kdenowcore
+    EXPORT_FILE_NAME kdenowcore_export.h
+    EXPORT_MACRO_NAME KDENOWCORE_EXPORT
+)
+
 set_target_properties(
-    kdenow PROPERTIES ENABLE_EXPORTS ON
+    kdenowd PROPERTIES ENABLE_EXPORTS ON
 )
 
-install(TARGETS kdenow ${INSTALL_TARGETS_DEFAULT_ARGS})
+target_include_directories(kdenowcore PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
+
+install(TARGETS kdenowd ${INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS kdenowcore EXPORT kdenowLibraryTargets
```

Comment:
Please make sure it exports cmake file, which one can be included in other projects for using kdenowcore.